nibabel/streamlines/tractogram.py

```diff
@@ -1,9 +1,14 @@
 import copy
 import numbers
 import numpy as np
-import collections
 from warnings import warn
 
+try:
+    from collections.abc import MutableMapping
+except ImportError:
+    #PY2 compatibility
```

Comment:
For PEP8, there should be a space after the `#`.

Reply:
Fixed